dev-client/src/screens/SoilScreen/TextureGuideScreen.tsx

```diff
@@ -102,6 +103,33 @@ export const TextureGuideScreen = (props?: SoilPitInputScreenProps) => {
       <ScrollView bg="grey.300">
         <Column space="16px">
           <Column p="15px" bg="primary.contrast">
+            <Text variant="subtitle1">{t('soil.texture.guide.prepare')}</Text>
+            <BulletList
+              data={[1, 2, 3]}
+              renderItem={i => (
+                <Text variant="body1" color="text.primary">
+                  <Trans i18nKey={`soil.texture.guide.prepare_details_${i}`} />
```

Comment:
FYI (or Suggestion if the design won't be changing): 
If you only have text styled in the same way, and aren't trying to say stuff like "part of this text will be styled this other way" or "we're gonna plop another component into a section of this text", you can just use the standard t() in the `<Text>` component, without needing the `<Trans>` component too.
```suggestion
                  {t(`soil.texture.guide.prepare_details_${i}`)}
```

For here and other places (except the grittyness one)

Reply:
I'm going to leave it as-is for now. Hopefully we can fix this with a better BulletList component in the future.